lib/react_client/react_interop.dart

```diff
@@ -411,6 +411,8 @@ class ReactDartInteropStatics2 implements ReactDartInteropStatics {
     )
         handleComponentDidUpdate,
     void Function(Component2 component) handleComponentWillUnmount,
+    void Function(Component2 component, dynamic error, dynamic info) handleComponentDidCatch,
+    JsMap Function(ComponentStatics2 componentInstance, dynamic error) handleGetDerivedStateFromError,
```

Comment:
`componentInstance` seems like a misleading name since the instance is actually a field within the `ComponentStatics2` instance being passed in.